addons/hashes/fnc_hashEachPair.sqf

```diff
@@ -44,14 +44,11 @@ private ["_keys", "_values"];
 _keys = _hash select HASH_KEYS;
 _values = _hash select HASH_VALUES;
 
-for "_i" from 0 to ((count _keys) - 1) do
 {
-    private ["_key", "_value"];
-
-    _key = _keys select _i;
-    _value = _values select _i;
+    private _key = _x;
+    private _value = _values select _forEachIndex;
     TRACE_2("VM CHECK",_key,_value);
     call _code;
-};
+} forEach _keys;
```

Comment:
It adds `_forEachIndex` to the "data passed to the function on each iteration", right?